addon/styles/_frost-guage.scss

```diff
@@ -0,0 +1,33 @@
+.frost-guage {
+  position: relative;
+
+  &-text {
```

Comment:
Actually, I'm full of **** - BEM endorses `--` for modifiers, so I'm totally off base with peer classes.